packages/@sanity/portable-text-editor/src/types/editor.ts

```diff
@@ -88,7 +88,11 @@ export interface History {
 /** @beta */
 export type EditorSelectionPoint = {path: Path; offset: number}
 /** @beta */
-export type EditorSelection = {anchor: EditorSelectionPoint; focus: EditorSelectionPoint} | null
+export type EditorSelection = {
+  anchor: EditorSelectionPoint
+  focus: EditorSelectionPoint
+  backward?: boolean
```

Comment:
I wonder if it would be a bit more clear with `direction: "forward" | "backward"` – what do you think?

Reply:
It would be clearer, but then it would also have to be required imo.

The absence of "backward" means it's forward by default.